tools/go.mod

```diff
@@ -1,9 +1,197 @@
 module github.com/lightninglabs/faraday/tools
 
-go 1.16
+go 1.21
```

Comment:
We might as well bump this version too...

Reply:
Done.